public/index.html

```diff
@@ -4,10 +4,10 @@
     <title>Hello React</title>
     <!-- Not present in the tutorial. Just for basic styling. -->
     <link rel="stylesheet" href="css/base.css" />
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.12.2/react.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.12.2/JSXTransformer.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/showdown/0.3.1/showdown.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/react/0.13.0/react.js"></script>
+    <script src="https://cdn.jsdelivr.net/react/0.13.0/JSXTransformer.js"></script>
+    <script src="https://cdn.jsdelivr.net/jquery/2.1.3/jquery.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/showdown/0.3.1/showdown.js"></script>
```

Comment:
Oooo, looks like I can get fancy and combine all of these into a single request:

```html
<script src="https://cdn.jsdelivr.net/g/react@0.13.0(react.js+JSXTransformer.js),jquery(jquery.min.js),showdown"></script>
```

It doesn't actually do much for us here but it is *fancy*